extensions/amp-analytics/amp-analytics.md

```diff
@@ -836,6 +838,12 @@ Referrer policy is only available for `image` transport. If `referrerPolicy: no-
 }
 ```
 
+#####  Warning Message
```

Comment:
I'm a bit hesitated to document 'warningMessage' here because it's vendor specific. What do you think about https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md